### ISSUE TITLE
NTR: Dihydroxyacetone phosphate acetyltransferase low in fibroblast (Lab: pathology: biopsies & morphology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9287]
New term request Dihydroxyacetone phosphate acetyltransferase low in fibroblast (segal_070726205233): DHAP-AT

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000426
- label: Reduced Acyl-CoA:dihydroxyacetone phosphate acyltransferase activity in cultured fibroblasts
- parents: [Abnormal enzyme activity in cultured fibroblasts (HP:4000195)](https://hpo.jax.org/app/browse/term/HP:4000195)
- pmid: PMID:11152660
- def: Acitivity of acyl-CoA:dihydroxyacetone phosphate acyltransferase below the lower limit of normal in cultured fibroblasts.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9287
- synonyms:
